src/routes/user.router.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const router = express.Router();
+
+const CartController = require("../controllers/cart.controller");
+const WishlistController = require("../controllers/wishlist.controller");
+const UserController = require("../controllers/user.controller");
+
+const { verifyAuth } = require("../middlewares/verifyAuth");
+
+router.get("/", UserController.getAllUsers);
```

Comment:
Why not use `verifyAuth` middleware for getAllUsers? Is it a public endpoint?

Reply:
Resolved.